watchtower/wtwire/features.go

```diff
@@ -9,6 +9,8 @@ var FeatureNames = map[lnwire.FeatureBit]string{
 	AltruistSessionsOptional: "altruist-sessions",
 	AnchorCommitRequired:     "anchor-commit",
 	AnchorCommitOptional:     "anchor-commit",
+	TaprootCommitRequired:    "taproot-commit",
+	TaprootCommitOptional:    "taproot-commit",
```

Comment:
Perhaps add the `staging` qualification like we did for the actual channel type? So far there's a slight (mostly inconsequential) script change on deck.

Reply:
> Do you just mean in this string or everywhere in this diff? Asking cause this string:

I mean to distinguish from the current taproot channels we've implemented to w/e version gets in once everyone else has implemented it. I think it matters most in the DB, and also on the wire w.r.t knowing which blob type matches up with a given channel type. 

Returning to this again though, maybe we don't actually need to distinguish? The current changes are just some small script changes, but the control block and such is given to the tower in an opaque manner, so all it needs to do is set the proper fields in the transaction and broadcast 🤔